released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.1.1.0/opam

```diff
@@ -0,0 +1,67 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-C" "theories" "-j%{jobs}%"]
+install: [make "-C" "theories" "install"]
+depends: [
+  "coq-mathcomp-classical" { = version}
+  "coq-mathcomp-solvable" { (>= "2.0.0") }
+  "coq-mathcomp-field"
+  "coq-mathcomp-bigenough" { (>= "1.0.0") }
+]
+
+tags: [
+  "category:Mathematics/Real Calculus and Topology"
+  "keyword:analysis"
+  "keyword:extended real numbers"
+  "keyword:filter"
+  "keyword:Cantor"
+  "keyword:topology"
+  "keyword:real numbers"
+  "keyword:sequence"
+  "keyword:convexity"
+  "keyword:Landau notation"
+  "keyword:logarithm"
+  "keyword:sin"
+  "keyword:cos"
+  "keyword:tangent"
+  "keyword:trigonometric function"
+  "keyword:exponential"
+  "keyword:differentiation"
+  "keyword:derivative"
+  "keyword:measure theory"
+  "keyword:integration"
+  "keyword:Lebesgue"
+  "keyword:probability"
+  "logpath:mathcomp.analysis"
+  "date:2024-03-31"
+]
+authors: [
+  "Reynald Affeldt"
+  "Alessandro Bruni"
+  "Yves Bertot"
+  "Cyril Cohen"
+  "Marie Kerjean"
+  "Assia Mahboubi"
+  "Damien Rouhling"
+  "Pierre Roux"
+  "Kazuhiko Sakaguchi"
+  "Zachary Stone"
+  "Pierre-Yves Strub"
+  "Laurent Théry"
+]
+url {
+  src: "https://github.com/math-comp/analysis/archive/1.1.0.tar.gz"
```

Comment:
Note that these links are no longer guaranteed to work (since Feb 2024), that is, they are generated on the fly and their checksum can vary. You should download them and reupload them as assets, to actually freeze them.

I will merge anyway, but you should keep that in mind.

Reply:
Can this be solved instead by using the url `https://github.com/math-comp/analysis/archive/refs/tags/1.1.0.tar.gz`?

Reply:
No, since the archives are generated on the fly, the risk is that github change the way it generates them, whatever the precise link used.

Reply:
Not really. The main issue is that Github does not want to promise that they will never upgrade their archive compressor (or change its compression options). So, since they completely broke the Internet one year ago by doing so, they had to revert their changes and announced that would refrain from changing their compressor for one whole year, so that people would have enough time to change the way they store archives. But this one-year reprieve has now expired and it will not be long before they break the Internet again.

Reply:
Off-topic pro-nix sidenote: nix uses the hash of the uncompressed archive, which is more stable